### DISCUSSION TITLE
Ask a question

### DISCUSSION THREAD

**Discussion by xiaohong2022** [Tampermonkey/tampermonkey#1655] [Q&A] [answered]
I would like to ask where is the download link of the BETA crx file of tampermonkey?

Chrome browser's extension store can't be opened in China.

**Comment by 7nik** [comment, marked as answer]
https://tampermonkey.net/crx/tampermonkey_beta_current.crx though not sure how permanent it is

**Reply by xiaohong2022** [reply]
Thanks a lot
